docs/development/testguide.rst

```diff
@@ -237,8 +237,8 @@ commandline option.  For example, to use 4 processes::
 
     python setup.py test --parallel=4
 
-Pass a negative number to ``'--parallel'`` to create the same number of
-processes as cores on your machine.
+Pass ``'--parallel=auto'`` to create the same number of processes as
```

Comment:
Shouldn't it be ``--parallel='auto'``? (Note the placement of first quote)

Reply:
Actually, no quote is needed.